### PR TITLE
[wip] Support notebook 4.2

### PR DESCRIPTION
**Pull request by bollwyvl**
This updates the metapackage.

Need to add tests.

Won't be able to fully test until we have upstreams in the `dev` label.

### CODE REVIEW

File: examples/nb_gist/README.md [hunk 1]
```diff
@@ -1,7 +0,0 @@
-# NBExtension example
```
Comment (damianavila) [inline examples/nb_gist/README.md]: Nice! this nb_gist is a leftover that should be removed...

File: conda.recipe/meta.yaml [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
 
 requirements:
   build:
     - python
   run:
     - python
-    - ipyparallel
-    - ipywidgets
-    - pandoc
```
Comment (damianavila) [inline conda.recipe/meta.yaml]: this is derived from ipython-we, the customized thing we have on AEN, it makes sense to delete it in this context, I agree...

**Comment by damianavila** [comment]
LGTM

**Comment by bollwyvl** [comment]
Started working on tests here... I am hoping I can re-use the test machinery that ships with each of the upstreams, but I'll definitely have to do some patching, as, for example, nb_conda_kernels breaks the baseline machinery....

**Pull request merged**